library/packages/src/lib/y2packager/new_repository_setup.rb

```diff
@@ -0,0 +1,52 @@
+# ------------------------------------------------------------------------------
+# Copyright (c) 2022 SUSE LINUX GmbH, Nuremberg, Germany.
+#
+# This program is free software; you can redistribute it and/or modify it under
+# the terms of version 2 of the GNU General Public License as published by the
+# Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+# ------------------------------------------------------------------------------
+
+require "yast"
+
+require "singleton"
+
+module Y2Packager
+  #
+  # This class stores the new repositories and services added during
+  # installation or upgrade. It can be used together with the
+  # OriginalRepositorySetup class to find the old and new repositories.
```

Comment:
if that comparison will be made on multiple places, then maybe it makes sense to have class that find that diffs also in yast2?

Reply:
It is just a trivial "exclude", see https://github.com/yast/yast-installation/pull/1015/files#diff-d90a84d467a4dedd884dbed8f308018c56b095a7423a00109b558129a3ec8703R70